.circleci/config.yml

```diff
@@ -198,17 +132,13 @@ jobs:
     - run:
         name: Linting
         command: |
-          gometalinter --disable-all --vendor --deadline=60s \
-            --enable=golint \
-            --enable=vet \
-            ./...
+          # TODO(gbbr): linting
+          # go get -v -u golang.org/x/lint/golint
+          # golint `go list ./...` | grep -v globalsign/mgo/collection.go
 
     - run:
         name: Testing
         command: |
-              mkdir /tmp/test-results
-              INTEGRATION=1 gotestsum --junitfile /tmp/test-results/result.xml -- -race `go list ./...`
-    - store_artifacts:
-        path: /tmp/test-results
-    - store_test_results:
-        path: /tmp/test-results
+              INTEGRATION=1 go test -race `go list ./... | grep -v grpc.v12`
+              go get google.golang.org/grpc@v1.2.0
+              go test ./contrib/google.golang.org/grpc.v12/...
```

Comment:
Hmm... Google broke compatibility within a major version, so there's not much else we can do here I think.

Reply:
Me too, but don't think there's any other better way.